src/tactic/mk_opaque.lean

```diff
@@ -0,0 +1,21 @@
+import tactic.core
+
+namespace tactic
+
+setup_tactic_parser
+
+/-- `mk_opaque x y z`, with `x`, `y`, `z` local definitions, transforms them into
+normal local constants -/
+meta def interactive.mk_opaque (ns : parse ident*) : tactic unit :=
```

Comment:
This should be a separate PR; it also needs more docs

Reply:
This is #3700.